smart-contract-extensions/test/hooks/codeRequiredHook.js

```diff
@@ -81,6 +91,27 @@ contract('CodeRequiredHook', accounts => {
     )
   })
 
+  describe('can remove codes', () => {
+    beforeEach(async () => {
+      await hookContract.removeCodes(lock.address, [answerAccount.address], {
```

Comment:
Minor, maybe change `answerAccount` to `codeAccount` for consistency?

Reply:
yup good point